test/query.test.js

```diff
@@ -1401,6 +1401,9 @@ describe('query', () => {
   })
 
   test('containsstr', () => {
+    expect(Expr.toString(query.ContainsStr('ABC', 'A'))).toEqual(
```

Comment:
are we going to move these type of assertions close to the function test? I'm saying because there is a test specific for this case